### PR TITLE
Added Netlify Github Action and one readme change.

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Jonsy13 <vedant.shrotria@chaosnative.com>

**What this PR does / why we need it**:

**This PR will add -**
**- GitHub Action for Netlify Live Deploy.** 
**- some refactoring in Action workflows and Readme.md.**
Test-Run - https://github.com/Jonsy13/test-netlify-action/runs/2735288763

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [X] Fixes #<issue number>
-   [X] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

**Comment by Jonsy13** [comment]
Currently have added #DONOTMERGE.
I will test it once more and then update it here.

**Comment by Jonsy13** [comment]
This PR is ready now. 
Have added Github Action for Netlify Build.
The Auto-Build for this git-repo from netlify needs to be deactivated.
We need to add 2 Secrets - **NETLIFY_AUTH_TOKEN** and **NETLIFY_SITE_ID**
**[ Test Run Can be checked Here ] :** Jonsy13/test-netlify-action#2

File: website/docs/agent-installation.md [hunk 1]
```diff
@@ -11,13 +11,21 @@ In Litmus the Agents can be classified as two types
 
 As part of Litmus installation by default, a self cluster would be registered as Agent in the Portal. From Portal you induce chaos into self cluster and observe the results from the Portal.
 
+<<<<<<< HEAD
```
Comment (uditgaurav) [inline website/docs/agent-installation.md]: Looks like the conflict is not resolved properly @Jonsy13 do you want to take a look?

**Review by uditgaurav — CHANGES_REQUESTED** [review]

File: website/docs/agent-installation.md (hunk 1 above)
Comment (Jonsy13) [reply website/docs/agent-installation.md]: Sure, thanks. Let me check it.

**Comment by Jonsy13** [comment]
Test-Run - https://github.com/Jonsy13/test-netlify-action/runs/2735288763

**Review by uditgaurav — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**